build/common.sh

```diff
@@ -55,7 +55,11 @@ readonly KUBE_BUILD_PPC64LE="${KUBE_BUILD_PPC64LE:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat ${KUBE_ROOT}/build/build-image/cross/VERSION)"
-# KUBE_BUILD_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+# KUBE_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+
+# Increment this version to cause everyone to rebuild images, including the data
+# container.
```

Comment:
How often should we be doing this version change? My guess is major versions. Is this 5 for k8s v1.5?